src/metaswitch/clearwater/etcd_tests/etcdtestbase.py

```diff
@@ -35,9 +35,10 @@
 import sys
 import unittest
 from .etcdcluster import EtcdCluster
+from metaswitch.common.logging_config import configure_test_logging
+
```

Comment:
i'm not really fussed one way or the other. Just makes sense to keep one format throughout